<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsGui</class>
 <widget class="QWidget" name="SettingsGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="m_tabUi">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_tabIconSkin">
      <attribute name="title">
       <string>Icons &amp;&amp; skins</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="m_lblIconTheme">
         <property name="text">
          <string>Icon theme</string>
         </property>
         <property name="buddy">
          <cstring>m_cmbIconTheme</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="m_cmbIconTheme"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_lblStyle">
         <property name="text">
          <string>Style</string>
         </property>
         <property name="buddy">
          <cstring>m_cmbStyles</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="m_cmbStyles"/>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="m_checkForceAlternativePalette">
         <property name="text">
          <string>Use skin colors</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="HelpSpoiler" name="m_helpSkinColors" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="m_lblSkin">
         <property name="text">
          <string>Skin</string>
         </property>
         <property name="buddy">
          <cstring>m_treeSkins</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTreeWidget" name="m_treeSkins">
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <attribute name="headerMinimumSectionSize">
          <number>80</number>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>643</width>
           <height>97</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabAppFont">
      <attribute name="title">
       <string>Application font</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="0">
        <widget class="QGroupBox" name="m_gbAppFont">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Customize global font</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="m_lblAppFont">
            <property name="text">
             <string>Font preview</string>
            </property>
            <property name="buddy">
             <cstring>m_btnChangeFont</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="m_btnChangeFont">
            <property name="text">
             <string>&amp;Change font</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="m_checkFontAntialiasing">
         <property name="text">
          <string>Enable application-wide font anti-aliasing</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabCustomSkinColors">
      <attribute name="title">
       <string>Custom skin colors</string>
      </attribute>
      <layout class="QFormLayout">
       <item row="1" column="0" colspan="2">
        <widget class="HelpSpoiler" name="m_helpCustomSkinColors" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="m_gbCustomSkinColors">
         <property name="title">
          <string>Customize predefined colors</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="m_layoutCustomColors"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabNotifications">
      <attribute name="title">
       <string>Tray area</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="1" column="0">
        <widget class="QGroupBox" name="m_grpTray">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Tray icon</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldGrowthPolicy::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="m_checkMonochromeIcons">
            <property name="text">
             <string>Use monochrome icon</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="m_checkCountUnreadMessages">
            <property name="text">
             <string>Display count of unread messages</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="m_checkHideWhenMinimized">
            <property name="text">
             <string>Hide main window when it is minimized</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="m_checkColoredIconsWhenArticles">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Use colored icon when there are unread articles</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="m_checkHidden">
         <property name="text">
          <string>Start application minimized (or hidden if configured)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabTaskBar">
      <attribute name="title">
       <string>Task bar</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QCheckBox" name="m_displayUnreadMessageCountOnTaskBar">
         <property name="text">
          <string>Display count of unread messages</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabTabs">
      <attribute name="title">
       <string>Tabs</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="m_grbCloseTabs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Close tabs with</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="m_checkCloseTabsMiddleClick">
            <property name="text">
             <string>Middle mouse button single-click</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_checkCloseTabsDoubleClick">
            <property name="text">
             <string>Left mouse button double-click</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="m_checkNewTabDoubleClick">
         <property name="text">
          <string>Open new tabs with left mouse button double-click on tab bar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="m_checkHideTabBarIfOneTabVisible">
         <property name="text">
          <string>Hide tab bar if just one tab is visible</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabToolBars">
      <attribute name="title">
       <string>Toolbars</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_18">
       <item row="0" column="1">
        <widget class="QComboBox" name="m_cmbToolbarButtonStyle"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="m_cmbSelectToolBar">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Toolbar for feeds list</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Toolbar for articles list</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Statusbar</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QStackedWidget" name="m_stackedToolbars">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="m_pageFeedsToolbar">
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="ToolBarEditor" name="m_editorFeedsToolbar" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="m_pageMessagesToolbar">
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="ToolBarEditor" name="m_editorMessagesToolbar" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="m_pageStatusbar">
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="ToolBarEditor" name="m_editorStatusbar" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Button style</string>
         </property>
         <property name="buddy">
          <cstring>m_cmbToolbarButtonStyle</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Toolbar editor</string>
         </property>
         <property name="buddy">
          <cstring>m_cmbSelectToolBar</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="m_spinToolbarIconSize">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="singleStep">
          <number>8</number>
         </property>
         <property name="value">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Icon size</string>
         </property>
         <property name="buddy">
          <cstring>m_spinToolbarIconSize</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpSpoiler</class>
   <extends>QWidget</extends>
   <header>helpspoiler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolBarEditor</class>
   <extends>QWidget</extends>
   <header>toolbareditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_tabUi</tabstop>
  <tabstop>m_grpTray</tabstop>
  <tabstop>m_checkMonochromeIcons</tabstop>
  <tabstop>m_checkColoredIconsWhenArticles</tabstop>
  <tabstop>m_checkCountUnreadMessages</tabstop>
  <tabstop>m_checkHideWhenMinimized</tabstop>
  <tabstop>m_checkCloseTabsDoubleClick</tabstop>
  <tabstop>m_checkCloseTabsMiddleClick</tabstop>
  <tabstop>m_checkNewTabDoubleClick</tabstop>
  <tabstop>m_checkHideTabBarIfOneTabVisible</tabstop>
  <tabstop>m_cmbToolbarButtonStyle</tabstop>
  <tabstop>m_spinToolbarIconSize</tabstop>
  <tabstop>m_cmbSelectToolBar</tabstop>
  <tabstop>m_cmbIconTheme</tabstop>
  <tabstop>m_cmbStyles</tabstop>
  <tabstop>m_checkForceAlternativePalette</tabstop>
  <tabstop>m_treeSkins</tabstop>
  <tabstop>m_gbCustomSkinColors</tabstop>
  <tabstop>m_displayUnreadMessageCountOnTaskBar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
